frontend_tests/node_tests/people.js

```diff
@@ -491,19 +532,6 @@ run_test('multi_user_methods', () => {
 initialize();
 
 run_test('message_methods', () => {
-    const charles = {
-        email: 'charles@example.com',
-        user_id: 451,
-        full_name: 'Charles Dickens',
-        avatar_url: 'charles.com/foo.png',
-        is_guest: false,
-    };
-    // Maria is an intentionally incomplete user object without all attributes
```

Comment:
For example, this particular test lost that context about maria being incompolete. Adding `assert(maria.is_guest === undefined)` would return some of it to the data.

Reply:
added that comment back, although I feel even when the comment was originally added, it was a bit cryptic...it would have been nice to say what type of regression we were protecting against